sklearn/multiclass.py

```diff
@@ -731,7 +735,9 @@ class OutputCodeClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
         See :term:`Glossary <random_state>`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation.
+        The number of jobs to use for the computation: the code_size
+        Output-Code multiclass problems are computed in parallel.
```

Comment:
looks like the actual number is tricky here so let's just go with "the multiclass problems are computed in parallel."

Reply:
Hey @NicolasHug, I have modified accordingly. Can you please check the PR? Thanks!